components/at_mention/at_mention.test.tsx

```diff
@@ -15,19 +17,19 @@ describe('components/AtMention', () => {
         currentUserId: 'abc1',
         teammateNameDisplay: General.TEAMMATE_NAME_DISPLAY.SHOW_NICKNAME_FULLNAME,
         usersByUsername: {
-            currentuser: {id: 'abc1', username: 'currentuser', first_name: 'First', last_name: 'Last'},
-            user1: {id: 'abc2', username: 'user1', first_name: 'Other', last_name: 'User', nickname: 'Nick'},
-            'userdot.': {id: 'abc3', username: 'userdot.', first_name: 'Dot', last_name: 'Matrix'},
+            currentuser: {id: 'abc1', username: 'currentuser', first_name: 'First', last_name: 'Last'} as UserProfile,
+            user1: {id: 'abc2', username: 'user1', first_name: 'Other', last_name: 'User', nickname: 'Nick'} as UserProfile,
+            'userdot.': {id: 'abc3', username: 'userdot.', first_name: 'Dot', last_name: 'Matrix'} as UserProfile,
```

Comment:
can't we create valid Objects here instead of type casting what we need?

Reply:
i change this to use `TestHelper` methods here. why not using these methods in evey tests?

Reply:
I think some of the tests that aren't using it were tests were created before the TestHelper methods were added, and no one ever went back and updated the old tests.